src/util/states.ts

```diff
@@ -437,7 +438,10 @@ export function blurSeries(
 
     const blurredSeries: SeriesModel[] = [];
 
-    ecModel.eachSeries(function (seriesModel) {
+    ecModel.eachSeries(function (seriesModel, seriesIndex) {
+        if (excludedSeriesIndexes && excludedSeriesIndexes.includes(seriesIndex)) {
```

Comment:
`includes` is not supported in IE 9+, which we still supported currently. We can use `zrUtil.indexOf(excludedSeriesIndexes, seriesIndex) >= 0` instead.

Reply:
fixed